### PR TITLE
Add lesson-note divs to some places where applicable

### PR DESCRIPTION
**Pull request by 01zulfi**
Complete the following REQUIRED checkboxes:

-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/theodinproject/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`

Complete the following checkboxes ONLY IF they are applicable to your PR. You can complete them later if they are not currently applicable:
-   [ ] I have previewed all lesson files included in this PR with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure the Markdown content is formatted correctly
-   [ ] I have ensured all lesson files included in this PR follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

<hr>

**1. Because:**

- https://github.com/TheOdinProject/theodinproject/pull/3032

**2. This PR:**

Adds `.gold-border` divs where applicable

**3. Additional Information:**

n/a

### CODE REVIEW

**Comment by 01zulfi** [comment]
Recent commit based on https://github.com/TheOdinProject/theodinproject/pull/3032/commits/4d259807886205423425060b1627f8572a7aff8b

**Review by thatblindgeye — CHANGES_REQUESTED** [review]
Because these "notes" are now using a more stylized container, I don't think we really need to keep the "note:", "important note", etc. text.

File: javascript/react_js/router.md [hunk 1]
```diff
@@ -96,7 +96,11 @@ You should now have enough basics to get started with React routing. There are a
 1. Go and add a few new routes to the application we created above; playing around with it is the best practice. Consider deleting it completely and rewriting it for practice.
 2. Browse a little bit through the React Router [documentation](https://reactrouter.com/docs/en/v6/getting-started/overview). You don't need to read through all of it, nor understand all of it. Just browse through the concepts we discussed here and re-read them. This is a great resource to refer back to.
 3. Watch this [video on React Router by Dev Ed](https://www.youtube.com/watch?v=Law7wfdg_ls) for reviewing. You should already be familiar with those concepts.
-    *  **Note:** What we have learned in this lesson is the latest version of React Router (v6). However, in this video, an older version of React Router (v5) is used. The terminologies might be different, but the concept remains the same.
+    *  <div class="lesson-note" markdown="1">
```
Comment (thatblindgeye) [inline javascript/react_js/router.md:99]: Not a blocker so I'd say it's your call, but whether we need this bullet for the note or if we can just indent the `div` tags 4 spaces is worth a thought. Either way should work, though, so it's really if you find either way to be more visually pleasing/apparent.

**Review by thatblindgeye — APPROVED** [review]
🎉 left another comment below, but it's nothing blocking so you can feel free to merge if you'd like.

File: javascript/react_js/router.md (hunk 1 above)
Comment (01zulfi) [reply javascript/react_js/router.md:99]: I initially thought of that, but indenting the div doesn't render quite right:

image

Merging this PR for now. Later, I'll assess if we want a stylized div here at all.

**Pull request merged**